### PR TITLE
Home design frontend

### PR DESCRIPTION
**Pull request by EetuHei**
- Added links to login, register and game pages.

- Styling for the landing page, mobile responsive.

Check it out and let me know if you want different icons / text styling etc.

### CODE REVIEW

**Comment by diegocasillas** [comment]
Can you add some screenshots? I'll check it when I have time.

**Comment by EetuHei** [comment]
> Can you add some screenshots? I'll check it when I have time.

image

image

**Comment by EetuHei** [comment]
> Can you add some screenshots? I'll check it when I have time.

Forgot to add slight shadow on mobile, but fixed it.
Here is the screenshot with the shadow added:
image

File: client/src/routes/Home/index.module.css [hunk 1]
```diff
@@ -0,0 +1,146 @@
+body {
```
Comment (diegocasillas) [inline client/src/routes/Home/index.module.css]: Remove this file when it is not needed anymore.

**Pull request merged**